contracts/Allocator.sol

```diff
@@ -396,21 +435,21 @@ contract Allocator is Permissions, IERC777Recipient {
      */
     function calculateVestedAmount(address wallet) public view returns (uint256 vestedAmount) {
         ITimeHelpers timeHelpers = ITimeHelpers(contractManager.getContract("TimeHelpers"));
-        uint256 date = now;
         Beneficiary memory beneficiaryPlan = _beneficiaries[wallet];
         Plan memory planParams = _plans[beneficiaryPlan.planId - 1];
         vestedAmount = 0;
-        if (date >= timeHelpers.addMonths(beneficiaryPlan.startMonth, planParams.vestingCliff)) {
+        uint256 currentMonth = timeHelpers.getCurrentMonth();
+        if (currentMonth >= beneficiaryPlan.startMonth.add(planParams.vestingCliff)) {
```

Comment:
currentMonth - number of months from some date
beneficiaryPlan.startMonth - timestamp of the vesting plan (or not?)
planParams.vestingCliff - number of months when tokens fully locked
What is the result of adding timestamp to number of months?

Reply:
Not. `StartMonth` is also number of months from some date.
Everything is aligned over months so I think it's a source of confusion to store `startMonth` as timestamp.
And yes. It makes life a little bit easier in computations.

Reply:
Discussed